libs/full/parcelset_base/include/hpx/parcelset_base/parcelset_base_fwd.hpp

```diff
@@ -14,7 +14,7 @@
 namespace hpx::parcelset {
 
     class HPX_EXPORT parcelport;
-    class HPX_EXPORT locality;
+    class /* HPX_EXPORT */ locality;
```

Comment:
I think that the `HPX_EXPORT` should stay here (at the declaration). Instead, we should remove it here: https://github.com/STEllAR-GROUP/hpx/blob/master/libs/full/parcelset_base/include/hpx/parcelset_base/locality.hpp#L30 (at the definition). That also requires adding `#include <hpx/parcelset_base/parcelset_base_fwd.hpp>` to this file.

Reply:
I have applied this update.